src/components/ui/Animation.js

```diff
@@ -65,6 +65,7 @@ export default function animateChart(stx, animationParameters, easeMachine) {
         if (params !== undefined && params.animationEntry) return;
 
         function completeLastBar(value) {
+            if (!chart.masterData) { return; }
```

Comment:
@brucebinary when switching to historical data `masterData` get null, so that `.length`  throw an error, I put this piece of code for this reason.